source/usage-examples/command.txt

```diff
@@ -5,23 +5,33 @@ Run a Command
 .. default-domain:: mongodb
 
 You can run all raw database operations, not including
-:manual:`CRUD </crud/>` operations, using the :node-api:`command()
-<Admin.html#~command>` method. Call the ``command()`` method with
-your command object on an instance of ``Admin`` for diagnostic and
+:manual:`CRUD </crud/>` operations, using the :node-api:`db.command()
+<Db.html#~command>` method. Call the ``command()`` method with
+your command object on an instance of a database for diagnostic and
 administrative tasks such as fetching server stats or initializing a replica
 set.
 
 .. note::
     Use :manual:`collection methods </reference/method/js-collection>`
```

Comment:
Why is this linking to shell documentation?

Reply:
I'll update this note to advise the reader to use the mongo shell to run administrative commands. My guess is the author's intent was to convey this, but it could have been more clear.